### DISCUSSION TITLE
the annotation of the macs2 called peaks, Cell_type information

### DISCUSSION THREAD

**Discussion by ZunpengLiu** [GreenleafLab/ArchR#1816] [General]
Hello,

Thanks for developing and managing the fantastic software. It is very helpful!

I am calling the peaks for each cell type by macs2 following the manual [https://www.archrproject.com/bookdown/calling-peaks-w-macs2.html](url).

I export the peakSet  using `getPeakSet(proj)` function. And, I found that these peaks for all groups/cell types are not shared between groups/cell types. 

Yes, I do see your explanation about these "However, these annotations do not inherently mean that the given peak was only called in that group, rather that the annotated group had the highest normalized significance for that peak call." However, in some conditions, we actually want to compare all of the called/identified peaks among different cell types, for example the overlap/Jaccard analysis.

So, how can we get some more information about the detailed annotation for these identified/merged union peaks, not only the annotation for the group with highest significance. For example, for the Cell_type column , how can we get more groups, A, B, C, et al for the merged union peaks, or a binary column for each group to show whether the peak is exist in this specific group. 

Thanks! 

Zunpeng

**Comment by ZunpengLiu** [comment]
One solution in my mind is go back to the generated rds files for each group in PeakCalls folder. And, then do some overlap analysis based on these files. Do you think that is reasonable?

**Comment by rcorces** [comment]
If I'm understanding your question correctly, then this has been answered previously.
https://github.com/GreenleafLab/ArchR/search?q=PeakCalls&type=discussions

for ex https://github.com/GreenleafLab/ArchR/discussions/1546
